bears/python/MypyBear.py

```diff
@@ -0,0 +1,136 @@
+from collections import namedtuple
+import textwrap
+import sys
+import re
+import os
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+
+
+class FlagInfo(namedtuple('FlagInfo', 'arg doc inverse')):
+    """
+    Information about a command-line flag
+
+    :param arg:
+        The argument to pass to enable the flag
+    :param doc:
+        Help for the flag
+    :param inverse:
+        Set to true when the coala option is the inverse of
+        the subprocess one, for example coala's "allow_untyped_calls"
+        is the inverse of mypy's "--disallow-untyped-calls".
+    """
+
+    def want_flag(self, value):
+        """Check if the flag should be added to the argument list"""
+
+        if self.inverse:
+            value = not value
+        return value
```

Comment:
this was personal preference of the author, his opinion is that this is better readable